docs/content/policy-reference.md

```diff
@@ -859,6 +859,7 @@ The `request` object parameter may contain the following fields:
 | `timeout` | no | `string` or `number` | Timeout for the HTTP request with a default of 5 seconds (`5s`). Numbers provided are in nanoseconds. Strings must be a valid duration string where a duration string is a possibly signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". A zero timeout means no timeout.|
 | `tls_insecure_skip_verify` | no | `bool` | Allows for skipping TLS verification when calling a network endpoint. Not recommended for production. |
 | `tls_server_name` | no | `string` | Sets the hostname that is sent in the client Server Name Indication and that be will be used for server certificate validation. If this is not set, the value of the `Host` header (if present) will be used. If neither are set, the host name from the requested URL is used. |
+| `cache` | no | `boolean` | Cache HTTP response across OPA queries. Default: `false`. |
```

Comment:
Can we add a note here regarding which response headers this cares about? (Expires, max-age, I think?)

Reply:
👍 a small section below that explains how the cache invalidation works will be needed so that users implementing the server-side can set the headers accordingly.

Reply:
Added a note about this.

Reply:
Thanks!